Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.17-buster AS builder
+
+WORKDIR /build
+COPY . .
+
+RUN make build
+
+FROM gcr.io/distroless/static:debug
+
+WORKDIR /
+COPY --from=builder /build/bin/k8s .
+
+EXPOSE 8080
```

Comment:
It wasn't immediately clear to me why we'd need to expose 8080 here - exposing controller runtime's Prometheus metrics?

Reply:
yup, I believe they're enabled by default in controller-runtime.